Adding fullscreen button during gameplay

kaylina140 opened the pull request:
Put together a solution for #1507 - Added a fullscreen button to the in-game UI. Button appears in the top right corner next to the player stats and alternates between "fullscreen" and "contract" depending on the state of the screen. 


<a href="https://gitpod.io/#https://github.com/FreezingMoon/AncientBeast/pull/1651"><img src="https://gitpod.io/api/apps/github/pbs/github.com/kaylina140/AncientBeast.git/c554640fd50f267a1b2ea3c189452ecd08564d19.svg" /></a>

DreadKnight commented:
@kaylina140 I've did some testing. Here's some feedback:
- the text under it looks bad as it's not centered and doesn't have shadow, we're probably better off removing it altogether I think
- when in fullscreen mode already, the icon should change to the "contract" icon (has reversed icons), same as it does in the main menu; ideally the code should have been reused from the main menu (using text under during gameplay as well in that case), but that's probably a bit hard to do considering html imports, unless this is fully generated by some js script

kaylina140 commented:
Thanks so much for getting back to me so quickly!! I totally agree with your point on the text and went ahead and removed it. Fixed the issue with the icon not changing as well.

DreadKnight commented:
@kaylina140 I reply daily :-) Looks good now visually. The styling could be shared between the two buttons btw, so the related fullscreen button from `src/style/pre-match.less` should end up in `src/style/styles.less`, to keep things a bit more nice and tidy, but ideal it would be to reuse the whole button code.

kaylina140 commented:
I see what you mean. Condensed it to just styles.less.

DreadKnight commented:
@kaylina140 All good now, will merge the PR :+1: Don't forget to pick up your bounty :money_with_wings:  :D
